### ISSUE TITLE
Multi-level models

### ISSUE THREAD

**Issue by kylehamilton** [kylehamilton/MAJOR#13]
Currently there is a barebones set up for running multilevel models, this needs to be extended and made more functional.

**Comment by GabiGIS** [comment]
Dear Kyle 
Is there any change to get free-of-charge the more complete version of MAJOR, including multilevel models. 
Thanks a lot.
Kind regards,
Gabriel

**Comment by mcfanda** [comment]
I completely agree,  I see a lot of requests for this feature from jamovi users and meta-analysis students.
